libraries/AP_InertialSensor/AP_InertialSensor_SITL.cpp

```diff
@@ -76,13 +76,13 @@ void AP_InertialSensor_SITL::generate_accel(uint8_t instance)
     float zAccel = sitl->state.zAccel + accel_bias.z;
     const Vector3f &vibe_freq = sitl->vibe_freq;
     if (vibe_freq.is_zero()) {
-        xAccel += accel_noise * rand_float();
-        yAccel += accel_noise * rand_float();
+        //xAccel += accel_noise * rand_float();
```

Comment:
why remove ?

Reply:
This is another change that won't be merged.  In SITL the vibrations are applied equally on X, Y and Z axis but this is not what we see on real vehicle (normally Z is much higher than X and Y).

This PR's change is to add protection for high Z-axis vibration so we remove the X and Y vibes because they interfere with testing.